FND/Agreements/Contracts.rdf

```diff
@@ -479,7 +480,7 @@
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasContractDuration">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDuration"/>
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasTerm"/>
```

Comment:
has duration is the most general in our time ontology; hasTerm reflects the contract duration; I am ok with eliminating one or the other, but they are used in different places.  When I moved hasTerm, it was because it was the more general property, and is my preference over the more specific 'contract duration'.  I could deprecate hasContractDuration and replace it everywhere - if you are ok with that??